chalicelib/blueprints/announcements.py

```diff
@@ -0,0 +1,55 @@
+from chalice import Blueprint
+from chalicelib.services.ses import SesMailSender, SesDestination
+from chalicelib.services.mongo import MongoService
+import boto3
+
+announcements_routes = Blueprint(__name__)
+
+ses_client = boto3.client('ses')
+
+
+# why is GET needed?
+@announcements_routes.route("/announcement", methods=["GET", "POST"], cors=True)
+def send_announcement():
+    MongoServer = MongoService()
```

Comment:
I'd actually create a new directory called `chalicelib/modules` and add `mongo.py` and `ses.py` – I forgot to do this during the refactoring page sorry 😭 but essentially, `services` would handle the business logic and core functionalities, whereas modules would be more of the external microservices/technical logic handling!